content/en/security/cspm/findings.md

```diff
@@ -19,47 +19,33 @@ Cloud Security Posture Management is not available in this site.
+In addition to reviewing and responding to findings, you can set notifications for failed findings, and configure signals to correlate and triage misconfigurations in the same view as real-time threats generated by [Cloud SIEM][2] and [Cloud Workload Security][3]. This enables you to accelerate investigations as the root-cause for many of today's cloud breaches are misconfigured services that have been exploited by attackers.
 
 ## Findings
 
-A finding is the primary primitive for a rule evaluation against a resource. Every time a resource is evaluated against a rule, a finding is generated with a **Pass** or **Fail** status. Resources are evaluated in increments between 15 minutes and four hours (depending on type). Datadog generates new findings as soon as a new scan is completed, and stores a complete history of all findings for the past 15 months so they are available in case of an investigation or audit.
-
-{{< img src="security/cspm/findings/posture-management-overview.png" alt="An overview of the Posture Management Findings page" style="width:100%;">}}
-
-## Explore your cloud misconfigurations with Findings
-
-Clicking on an individual finding that has **failed** to see details about the misconfigured resource, the rule description, its framework or industry benchmark mapping, and suggested remediation steps.
-
-{{< img src="security/cspm/findings/signal-overview.png" alt="Failed signals in the side panel" style="width:65%;">}}
-
-Aggregate findings by rule using the query search bar. This view shows a checklist of all of the detection rules that Datadog scans. Filtering by `evaluation:fail` status narrows the list to all detection rules that have issues that need to be addressed. The side panel shows details of each resource that has been evaluated by the rule.
+A finding is the primary primitive for a rule evaluation against a resource. Every time a resource is evaluated against a rule, a finding is generated with a **Pass** or **Fail** status. Resources are evaluated in increments between 15 minutes and four hours (depending on type). Datadog generates new findings as soon as a scan is completed, and stores a complete history of all findings for the past 15 months so they are available in case of an investigation or audit.
 
-{{< img src="security/cspm/findings/evaluation-fail.png" alt="Filtering by evaluation fail" style="width:100%;">}}
+## Explore your cloud misconfigurations
 
-The side panel shows details of each resource that has been evaluated by the rule.
+Findings are displayed on the [Security Findings Explorer][1]. Aggregate findings by rule using the **Group by** filters and query search bar. For example, filtering by `evaluation:fail` narrows the list to all detection rules that have issues that need to be addressed. Findings can also be aggregated by resource to rank order resources that have the most failed findings so you can prioritize remediation.
```

Comment:
```suggestion
Findings are displayed on the [Security Findings Explorer][1]. Aggregate findings by rule using the **Group by** filters and query search bar. For example, filtering by `evaluation:fail` narrows the list to all detection rules that have issues that need to be addressed. Findings can also be aggregated by resource to rank resources that have the most failed findings so you can prioritize remediation.
```

the "order" looks like a typo to me--unless "order resources" are a thing? in which case ignore this